osi_detectedlandmark.proto

```diff
@@ -159,24 +165,48 @@ message CandidateSign
 }
 
 //
-// \brief A candidate for (a) detected supplementary sign(s) as estimated by the
+// \brief Candidates for all detected supplementary signs of one traffic sign
+// as estimated by the sensor.
+//
+message CandidateSupplementarySigns
```

Comment:
@carsten-kuebler I think I do not understand this. Why is the base_rmse not in the "CandidateSupplementarySign". The name "candidate_multiple_supplementary_signs " seems not intuitive as well as I would not create two messages which sound so similar like "CandidateSupplementarySign" and "CandidateSupplementarySigns".
Should we have a call for that?

Reply:
I agree, how would you call a list of supplementary signs -> sign forest or sign jungle. Both sounds colloquially.

We can have a call from 11:30. Is it ok?